arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -499,7 +499,6 @@
           {
             "name": "parameters",
             "in": "body",
-            "required": true,
```

Comment:
is body optional for PUT ?
@johanste - What do you think?
Wasnt able to conclude. A [stack overflow link](http://stackoverflow.com/questions/7323958/are-put-and-post-requests-required-expected-to-have-a-request-body) for reference